.do/deploy.template.yaml

```diff
@@ -0,0 +1,9 @@
+spec:
+  name: docker-stellar-core-horizon
+  services:
+  - dockerfile_path: Dockerfile
+    git:
+      branch: acharb-quickdeploy
+      repo_clone_url: https://github.com/stellar/docker-stellar-core-horizon.git
+    name: docker-stellar-core-horizon
+    run_command: /start --standalone
```

Comment:
🌟 I think deploying in ephemeral mode is fine as long as we're clear, especially because that's also the default mode of the image. DigitalOcean does support specifying a one database though, per button, and so it could be worthwhile using that in the future so that quick deployments of this image use a datasource that doesn't evaporate when the instance is stopped.

Reply:
ya that makes sense to me. Especially if we decide we want a quick deploy for prod and testnet. Similar to above I think if we decide to add this for more than standalone this could be part of that work